src/main/java/Staff.java

```diff
@@ -0,0 +1,21 @@
+
+// Staff class representing a staff, with fields name, address, phone, email and
+// salary
+public class Staff {
+   
+    public Staff(String staffID, String name, String address, String phone, String email, double salary) {
+        super();
```

Comment:
I think there is no need for this super() since every class by default inherits from an existing Object class and we don't need to explicitly mention super().

Reply:
done